fragment.go

```diff
@@ -688,12 +712,11 @@ func (f *fragment) importSetValue(columnID uint64, bitDepth uint, value uint64)
 // A bitmap can be passed in to optionally filter the computed columns.
 func (f *fragment) sum(filter *Row, bitDepth uint) (sum, count uint64, err error) {
 	// Compute count based on the existence row.
-	row := f.row(uint64(bitDepth))
+	consider := f.row(uint64(bitDepth))
 	if filter != nil {
-		count = row.intersectionCount(filter)
-	} else {
-		count = row.Count()
+		consider = consider.Intersect(filter)
```

Comment:
Isn't `intersectionCount` faster than separate `Intersect` and `Count`?

Reply:
Yes, but this change is necessary because of a bug with this `sum()` method. It wasn't taking the not-null bitmap (`row(bitDepth)`) into consideration when calculating sum. So if not-null was set to `0`, but the BSI rows still contained a non-zero value, that value would be included in the sum. Basically, we need `consider` later in the method. 

With all that said, there may be a way to leverage `instersectCount()` that I'm missing. Can you suggest a different approach?

Reply:
No, that makes sense.